src/games/strategy/triplea/Properties.java

```diff
@@ -677,5 +677,9 @@ public static boolean getControlAllCanalsBetweenTerritoriesToPass(final GameData
     return data.getProperties().get(CONTROL_ALL_CANALS_BETWEEN_TERRITORIES_TO_PASS, false);
   }
 
+  public static boolean getRollRocketsAfterSBR(final GameData data) {
+    return data.getProperties().get("Roll rocket damage after processing SBR", false);
```

Comment:
Seems that it is baked in as part of rule set. Question is which rule sets, and how do we define we are using those rule sets in the XML.